apps/omg_watcher/test/test_helper.exs

```diff
@@ -15,12 +15,3 @@
 ExUnit.configure(exclude: [integration: true, property: true, wrappers: true])
 ExUnitFixtures.start()
 ExUnit.start()
-
-{:ok, _} = Application.ensure_all_started(:httpoison)
-{:ok, _} = Application.ensure_all_started(:fake_server)
-
-Mix.Task.run("ecto.create", ~w(--quiet))
-Mix.Task.run("ecto.migrate", ~w(--quiet))
-
-{:ok, _} = Application.ensure_all_started(:briefly)
```

Comment:
please keep the `ensure_all_starteds` for briefly and erlexec. I have tried removing those, but it seems to behave weirdly in some configurations of test invocations. If this keeps bugging us, we can defer to another PR.